types/index.d.ts

```diff
@@ -1,24 +1,16 @@
-export type ImgixClientOptions = {
-  domain: string;
-  secureURLToken?: string | undefined;
-  useHTTPS?: boolean;
-  includeLibraryParam?: boolean;
-  libraryParm?: string;
-};
-
-interface _buildURLOptions extends Omit<ImgixClientOptions, 'domain'> {
-  domain?: string;
-  disablePathEncoding?: boolean;
-}
```

Comment:
Moved these to the bottom of the file. It felt like we've been intentionally placing them from the start so just how housekeeping here 🧹 .